webpack.js

```diff
@@ -9,12 +9,14 @@ webpackConfig.module.rules = [
     test: /\.scss$/,
     use: ['style-loader', 'css-loader', 'sass-loader'],
   },
+  /*
   {
     test: /\.(js|vue)$/,
     use: 'eslint-loader',
     exclude: /node_modules/,
     enforce: 'pre',
   },
+  */
```

Comment:
why was this commented out?

Reply:
eslint was disabled due to version clashes. It will be improved in one of the future releases.